src/third_party_builtin/tess2/sweep.C

```diff
@@ -582,8 +580,8 @@ static int CheckForIntersect( TESStesselator *tess, ActiveRegion *regUp )
 
     if( orgUp == orgLo ) return FALSE;    /* right endpoints are the same */
 
-    tMinUp = MIN( orgUp->t, dstUp->t );
-    tMaxLo = MAX( orgLo->t, dstLo->t );
+    tMinUp = std::min( orgUp->t, dstUp->t );
```

Comment:
Hmm...tha macros are using `>=` and `<=` and I think `std:min` and `std::max` use only `>` and `<`. For two literal numbers, the numerical result of `>` and `>=` should be the same. However, we're replacing macros with function calls here and so the  way the operands are handled can indeed change and produce different results when things like side-effects are considered. Just wanna make sure you've confirmed the *new* and old approaches are identical for this code block?

Reply:
I ran the regression suite, is that sufficient?